recipes/weblogic.rb

```diff
@@ -0,0 +1,42 @@
+# Copyright:: 2011-Present, Datadog
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include_recipe '::dd-agent'
+
+# Monitor weblogic
+#
+# Assuming you have 2 instances "test" and "prod",
+# one with and one without authentication
+# you need to set up the following attributes
+# node.datadog.weblogic.instances = [
+#   {
+#     :host => "localhost",
+#     :port => "9090",
+#     :name => "prod",
+#     :username => "username",
+#     :password => "secret"
+#   },
+#   {
+#     :host => "localhost",
+#     :port => "9091",
+#     :name => "test"
+#   }
+# ]
+
+datadog_monitor 'weblogic' do
+  instances node['datadog']['weblogic']['instances']
+  logs node['datadog']['weblogic']['logs']
+  action :add
+  notifies :restart, 'service[datadog-agent]' if node['datadog']['agent_start']
+end
```

Comment:
Is there a specific reason not to use the default integration template, as is used in many other integrations, eg. aerospike: https://github.com/DataDog/chef-datadog/blob/4c39143a77ba608d6e1265c9a06a9f04497d5cc7/recipes/aerospike.rb#L431-L438?

Using the default integration template makes it so you don't have to define parsing rules in a specific erb file, and makes the recipe future-proof (if a new option is added to the `weblogic` integration in a future Agent version, the recipe won't need to be updated to be able to use it).

Reply:
Sure I will change it

Reply:
Thanks, I did change the code